submissions/hannasyn/movie-seat-booking/index.html

```diff
@@ -0,0 +1,387 @@
+                <span class="schedule__circle">12</span>
+              </label>
+            </div>
+            <div class="schedule__time">
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="10.00">
+                <span class="schedule__clock">10.00</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="12.45" checked>
+                <span class="schedule__clock">12.45</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="15.50">
+                <span class="schedule__clock">15.50</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="17.00">
+                <span class="schedule__clock">17.00</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="18.00">
+                <span class="schedule__clock">18.00</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="18.40">
+                <span class="schedule__clock">18.40</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="19.25">
+                <span class="schedule__clock">19.25</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="20.30">
+                <span class="schedule__clock">20.30</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="21.20">
+                <span class="schedule__clock">21.20</span>
+              </label>
+              <label class="schedule__item">
+                <input class="schedule__input input-hidden" type="radio" name="time" value="22.00">
+                <span class="schedule__clock">22.00</span>
+              </label>
+            </div>
+            <button type="submit" class="button">Confirm</button>
+          </form>
+        </div>
+      </article>
+      <section class="hall">
+        <h2 class="hall__title">Pick your seat</h2>
+        <p class="hall__screen">Screen</p>
+        <form action="#" class="seats">
+          <div class="seats__row">
+            <div class="seats__column">
+              <label class="seats__item">
+                <input class="seats__input input-hidden" type="checkbox" name="seat" value="1">
```

Comment:
If I check this seat and seat from line 155, how you understand the difference between them with the same value?)

Reply:
Thank you for pointing! My bad, I fixed it)